tests/e2e/subscription_suite_test.go

```diff
@@ -36,11 +36,12 @@ var _ = Describe("Subscription Config Suite Test", func() {
 	type SubscriptionConfigTestCase struct {
 		operators.SubscriptionConfig
 		failureExpected *bool
+		stream string
```

Comment:
So the intention is to have in a single suite run test both streams?

If only one stream then it doesn't need to be added to the test case config.

Reply:
OH.. so each Entry in the table can access the vars from the cli?   I wasn't able to get that to work but happy to change it once I figure the HOW.. :)  Thanks for the review

Reply:
It is currently exposed but a refactoring is due. Wanted to move it out to another package like here https://github.com/openshift/oadp-operator/pull/710/files#diff-7520c0381b468591b4362f559e3043268715f1862cc6f9ac8fed25367be166e9